zenodo/modules/records/templates/zenodo_records/record_detail.html

```diff
@@ -27,19 +27,22 @@
 {%- from "zenodo_records/macros/accessright.html" import accessright_label -%}
 {%- from "zenodo_records/macros/objecttype.html" import objecttype_label -%}
 {%- from "zenodo_records/macros/altmetric.html" import almetric_badge -%}
-{%- from "zenodo_records/macros/authors.html" import authors -%}
+{%- from "zenodo_records/macros/authors.html" import authors as authors_macro -%}
```

Comment:
But is is necessary to set the variable name, when you can just use ``records.creators`` instead?